ApiGateway/Models/SignalOutput.cs

```diff
@@ -0,0 +1,14 @@
+namespace MRS.ApiGateway.Models;
+
+using System.Runtime.Serialization;
+
+public enum SignalOutput
```

Comment:
Add `[Serializable]` to get rid of the warning, can go onto anything that's going to/from JSON or MQTT

Reply:
Done